ci/env/install-bazel.sh

```diff
@@ -95,6 +95,10 @@ fi
 
 bazel --version
```

Comment:
If this line works then this means it's part of path already? Seems our installation target is `bin/` folder?

Reply:
No, checkout the issue linked; I'm interested in this issue since I met the same issue when I was using github codespace.

Reply:
Executable accessible could be installed in the current directory.

Reply:
Don't we always install to ` target="$HOME/bin/bazel"` or `target="/bin/bazel"`

Reply:
For me particularly, 
```
@SaSikun ➜ /workspaces/ray (master) $ echo $HOME
/home/codespace

+ bazel --version
bazel 6.5.0
+ which bazel
/home/codespace/bin/bazel
```
And bazel does install under `$HOME/bin/bazel`, but it's not in `PATH`